api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -195,7 +197,44 @@ public void shouldCorrectlyActivateANewClaim() throws JsonProcessingException, N
         Optional<PaymentCycle> paymentCycle = repositoryMediator.getOptionalPaymentCycleForClaim(claim);
         assertThat(paymentCycle).isNotEmpty();
         assertThat(paymentCycle.get().getPaymentCycleStatus()).isEqualTo(PaymentCycleStatus.NEW);
+    }
+
+    /**
+     * Run through a lifecycle where an active claim becomes ineligible due to not being on a qualifying benefit.
+     * The claim will go from active to pending expiry, after 16 weeks the claim will become expired and their card
+     * status will be set to {@link CardStatus#SCHEDULED_FOR_CANCELLATION}.
+     *                                                     | one year
+     * |...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|
+     * | active claim becomes ineligible (due to not being on qualifying benefit). Claim status set to pending expiry, card status set to pending_cancellation
+     * |...............| Claim status set to expired. Card status set to scheduled_for_cancellation
+     */
+    @Test
+    @Disabled("HTBHF-2656")
+    @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
+    void shouldProcessClaimWhoBecomesIneligibleDueToNoQualifyingBenefit() throws JsonProcessingException, NotificationClientException {
+        LocalDate expectedDeliveryDate = LocalDate.now().plusWeeks(25);
+        UUID claimId = applyForHealthyStartAsPregnantWomanWithNoChildren(expectedDeliveryDate);
+        assertFirstCyclePaidCorrectly(claimId);
 
+        // Claim becomes ineligible
+        wiremockManager.stubIneligibleEligibilityResponse();
+        ageByOneCycle();
+        Claim claim = repositoryMediator.loadClaim(claimId);
+        assertThat(claim.getClaimStatus()).isEqualTo(PENDING_EXPIRY);
+        assertThat(claim.getCardStatus()).isEqualTo(CardStatus.PENDING_CANCELLATION);
+
+        // 16 weeks pass
+        LocalDateTime now = LocalDateTime.now();
```

Comment:
I got confused by what `LocalDateTime.now()` had to do with 16 weeks passing until I realised that it had nothing to do with it, could this comment be next to the aging method and `now` be nearer to the assertions where it is relevant?

Reply:
Swapped the order of LocalDateTime.now() and ageByNumberOfCycles.